test/bigtest/tests/note-view-test.js

```diff
@@ -49,6 +52,17 @@ describe('Note View page', () => {
         expect(noteView.referredRecord.itemBarcode).to.equal(`${request.item.barcode}`);
       });
 
+      describe('when click on edit button', () => {
+        beforeEach(async () => {
+          await noteView.clickEditButton();
+          await noteEdit.when(() => noteEdit.isPresent);
```

Comment:
I don't think `await noteEdit.when(() => noteEdit.isPresent);` is necessary. We only want to check that the url was changed correctly